megatron/core/parallel_state.py

```diff
@@ -527,7 +527,7 @@ def initialize_model_parallel(
     order: str = "tp-cp-ep-dp-pp",
     get_embedding_ranks: Optional[Callable[[List[int], Optional[int]], List[int]]] = None,
     get_position_embedding_ranks: Optional[Callable[[List[int], Optional[int]], List[int]]] = None,
-    create_gloo_process_groups: bool = True,
+    create_gloo_process_groups: bool = False,
```

Comment:
this is a major change that affect all training, it will increase the mem footprint, what's the reason?